index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
Wow!  That's really weird.  The docs don't indicate that it's not compatible across platforms.

Could you tell me what `path.sep` evaluates to a windows machine?  I'd assume it should be `\\`?